grid/social-media/posts-lookup/maps/twitter.suma

```diff
@@ -91,19 +93,36 @@ operation MapTweetToPost {
   tweet = args.tweets[index]
   includes = args.includes
   attachments = []
+  replyReference = undefined
+  replyId = undefined
+  author = undefined
 
-   set if(tweet && tweet.attachments && tweet.attachments.media_keys && includes.media) {
+  set if(tweet.attachments && tweet.attachments.media_keys && includes.media) {
     attachments = call foreach(mediaKey of tweet.attachments.media_keys) MapMediaAttachment(
         mediaKey = mediaKey,
         media = includes.media
       )
   }
 
+  set if(tweet.referenced_tweets) {
+    replyReference = tweet.referenced_tweets.find(referencedTweet => { return referencedTweet.type === 'replied_to'})
+  }
+
+  set if(replyReference !== undefined) {
+    replyId = replyReference.id
+  }
+
+  set if(tweet.author_id && includes.users) {
+    author = call MapUser(user = includes.users.find(user => { return user.id === tweet.author_id}))
```

Comment:
Same here; also in the unlikely (but possible?) case you don't find the user, MapUser should return `undefined`

Reply:
👍  fixed the edge case for not found author_id in includes